packages/test/bin/tfm-test.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+const commander = require('commander');
+const { version } = require('../package.json');
```

Comment:
It is odd that the linting didn't give an error here but you should import relative files after importing libraries.